### ISSUE TITLE
[FEAT]: Exclude folder or external unit from scrapper 

### ISSUE THREAD

**Issue by Fedess1986** [nova-video-player/aos-AVP#1165]
### Description

Would be great to have an option to exclude a folder (and subfodler) or external unit (USB HDD, flash drive, etc) from library scan.

Sometimes I connect to my AndroidTV an external unit (specifically external HDD) with tons of home made pictures and videos, I don't want it to scan al those thousands files. I just want to navigate folders structure to reach the file.

**Comment by jdrch** [comment]
Same request here. The current functionality either displays all the video files on your phone (not helpful) or it allows browsing 1 directory at a time, which apps like MiXplorer already do with thumbnails and internal players.

**Comment by courville** [comment]
What about adding a .nomedia file in the folder not to be indexed as proposed in nova FAQ https://home.courville.org/nova_video_player-faq/index.html ?

**Comment by jdrch** [comment]
> What about adding a .nomedia file in the folder not to be indexed as proposed in nova FAQ https://home.courville.org/nova_video_player-faq/index.html ?

That's problematic as the fact that I don't want *Nova* to scan a folder does *not* mean I want Android to ignore that folder globally.

For example, let's say I want to use Nova to watch and organize OTA TV downloads that are in `Folder A`. That means I don't want media from any other folders to show up in Nova. It does NOT mean I want Android to ignore those other folders, as that would render stuff like birthday party videos, etc. invisible.

In other words, the *status quo* basically renders Nova impractical for video library viewing and renders its primary differentiating feature vs. (for example) VLC useless.

I haven't been a software developer in a long time but it should be trivial to add a setting to tell Nova to scan user defined folders only. This is especially the case as Nova can already browse/traverse Android common storage within its UI anyway.

**Comment by courville** [comment]
Fair point (not your comment about triviality though). In latest github releases, you can go in a specific folder for local storage and hit on Android TV the (-) button at the top that should disable folder indexing.

**Comment by jdrch** [comment]
> Fair point (not your comment about triviality though). In latest github releases, you can go in a specific folder for local storage and hit on Android TV the (-) button at the top that should disable folder indexing.

Thanks, but I'm using this on a phone and don't want to use the Android TV UI. However, the latest version of VLC fixed its longstanding SMB folder media library bug, thus enabling me to manage library folders across multiple devices. As such, I'll be switching back to it.
